README.md

```diff
@@ -664,6 +666,37 @@ In *AWS CNI Mode* (`target-access-mode=AWSCNI`) the controller actively manages
 | `AWSCNI`    |   `false`   |  `true`  | PodIP != HostIP: limited scaling and host bound |
 | `AWSCNI`    |   `false`   | `false`  | free scaling, pod VPC CNI IP used               |
 
+## Advanced Options for NLBs
+
+### Extra Listen Ports
+
+Some real world scenarios may require non-standard TCP or UDP ingresses. The `zalando.org/aws-nlb-extra-listeners`
+annotation allows you to specify a list of additional listeners to add to your NLB. The value of the annotation should
+be a valid JSON string of the following format.
+
+```json
+[
+    {
+        "protocol": "TCP",
+        "listenport": 22,
+        "targetport": 2222,
+        "podlabel": "application=ssh-service"
+    }
+]
+```
+
+The `podlabel` value is used to register targets in the target group associated with the listener. This depends on the
+AWS CNI Mode feature, where individual pods receive accessible IP addresses. The value is used to identify pods running
+in the same namespace as the ingress that will receive traffic from the load balancer.
```

Comment:
So the NLB would listen on `listenport` with `protocol` and forward traffic to `podlabel` selected pod IPs on port `targetport`.
If so, please add a line in the doc to make this more clear to the reader.

Reply:
Added